splunklib/searchcommands/internals.py

```diff
@@ -687,7 +669,6 @@ def _write_record(self, record):
         # We may be running under PyPy 2.5 which does not include the _json module
         _iterencode_json = JSONEncoder(separators=(',', ':')).iterencode
     else:
-        # Creating _iterencode_json this way yields a two-fold performance improvement on Python 2.7.9 and 2.7.10
```

Comment:
If we are not cleaning up the code which purports to support PyPy 2.5 - which we *definitely* do not support at this stage given that that version was released in 2015 - then we should reinstate this comment, otherwise the code does not make sense.

Ideally, this entire block would be refactored to  modern standards; importing the private implementation of the module by using the underscore name should really no longer be necessary.

Reply:
I tried to refactor entire block to modern standards by simply adding function with `json.dumps`, but there are some corner cases that this method doesn't cover and it causes one test to fail.

For the sake of the release, I just added the comment back for now.

Reply:
Sounds good.